backend/db/migrations/20171116200117-create-group.js

```diff
@@ -0,0 +1,26 @@
+export function up (queryInterface, Sequelize) {
+  return queryInterface.createTable('groups', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    name: {
+      allowNull: false,
+      type: Sequelize.STRING
+    },
+    createdAt: {
```

Comment:
I would add `defaultValue: Sequelize.fn('NOW')` to all the `createdAt` and `updatedAt` fields